psp/tests/models/test_load_models.py

```diff
@@ -7,14 +7,15 @@
 from psp.typings import X
 
 # We ran the test once to get the results and pasted here the results.
-# This way we can make sure the output doens't change.
+# This way we can make sure the output of previous models doesn't change.
```

Comment:
nice, so this confirms backwards compatibility

Reply:
Yeah basically. Whenever I make changes to the model (usually a new "version"), I also train a model, save it (`model_v7.pkl` in this PR), and note its output. This way I make sure that old models always behave the same. It's not perfect because we're just training 1 model with default parameters, but it's better than nothing!